subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/KubernetesReservationClient.kt

```diff
@@ -233,7 +230,7 @@ class KubernetesReservationClient(
         try {
             kubernetesClient.apps().deployments().withName(deploymentName).delete()
         } catch (t: Throwable) {
-            logger.critical("Delete deployment $deploymentName", t)
+            logger.warn("Failed to delete deployment $deploymentName", t)
```

Comment:
Do not consider that critical without failure
The warning will be enough or we should fail whole task at the end